lib/improver/database.py

```diff
@@ -232,23 +232,26 @@ class VerificationTable(SpotDatabase):
 
     """
 
-    def __init__(self, output, outfile, primary_dim, tablename, experiment_ID):
-        super(VerificationTable, self).__init__(output, outfile, tablename, primary_dim)
+    def __init__(self, output, outfile, tablename, primary_dim, experiment_ID,
```

Comment:
Should variable names be kept to lower_case_with_underscores?

Reply:
Again happy to change